roles/openshift_logging_eventrouter/templates/eventrouter-dc.j2

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  name: logging-eventrouter
+  labels:
+    component: eventrouter
+    logging-infra: eventrouter
+    provider: openshift
+spec:
+  selector:
+    component: eventrouter
+    logging-infra: eventrouter
+    provider: openshift
+  replicas: "{{ replicas }}"
+  template:
+    metadata:
+      labels:
+        component: eventrouter
+        logging-infra: eventrouter
+        provider: openshift
+      name: logging-eventrouter
+    spec:
+      serviceAccount: aggregated-logging-eventrouter
+      serviceAccountName: aggregated-logging-eventrouter
+{% if node_selector is iterable and node_selector | length > 0 %}
+      nodeSelector:
+{% for key, value in node_selector.iteritems() %}
+        {{ key }}: "{{ value }}"
+{% endfor %}
+{% endif %}
+      containers:
+        - name: kube-eventrouter
+          # TODO: reference rh registry once it gets there
+          image: gcr.io/heptio-images/eventrouter:v0.1
+          imagePullPolicy: IfNotPresent
+{% if (memory_limit is defined and memory_limit is not none and memory_limit != "")
+or (cpu_limit is defined and cpu_limit is not none and cpu_limit != "") %}
+          resources:
+            limits:
+{% if memory_limit is defined and memory_limit is not none and memory_limit != "" %}
+              memory: "{{ memory_limit }}"
```

Comment:
We should also set requests.memory to be the same as limits.memory similar to logging so the pod does not get killed off during memory starvation on the node